test/ctia/entity/vulnerability_test.clj

```diff
@@ -79,3 +80,37 @@
        (is (= 200 (:status response)))
        (is (= #{"CVE-2018-0007" "CVE-2018-0024" "CVE-2018-0031"}
               (set (map :title (edn/read-string (:body response))))))))))
+
+(deftest cpe-match-strings-paging-test
+  (helpers/fixture-ctia-with-app
+   (fn [app]
+     (helpers/set-capabilities! app "foouser" ["foogroup"] "user" all-capabilities)
+     (whoami-helpers/set-whoami-response app "45c1f5e3f05d0" "foouser" "foogroup" "user")
+     (let [{{get-store :get-store} :StoreService} (app->APIHandlerServices app)
+           {{conn :conn} :state} (get-store :vulnerability)
+           _ (es-helpers/load-file-bulk app conn "./test/data/indices/sample-cpe-250.json")
+           _ (Thread/sleep 1000)
```

Comment:
I am currently working on it ;-)
Do you see an exception in the logs when you remove the sleep?
I already solved this in ductile for queries to Elasticsearch with large bodies: https://github.com/threatgrid/ductile/pull/16, and I am currently handling a similar issue with the test suite.